api/data_refinery_api/views.py

```diff
@@ -152,21 +152,19 @@ class SearchAndFilter(generics.ListAPIView):
 
     """
 
-    # Only Experiments with processed objects are exposed
-    queryset = Experiment.processed_public_objects.annotate(samples_count=Count('samples')).all()
-
-    # For developing, you can uncomment this to expose everything.
-    #queryset = Experiment.objects.annotate(samples_count=Count('samples')).all()
-
     serializer_class = ExperimentSerializer
     pagination_class = LimitOffsetPagination
 
     filter_backends = (NoMarkupDjangoFilterBackend, filters.SearchFilter, filters.OrderingFilter)
     filter_class = ExperimentFilter
 
+    # Ordering
+    ordering_fields = ('total_samples_count', 'id', 'created_at',
+                       'source_first_published', 'accession_code',)
+    ordering = ('-total_samples_count',)
+
     # Ordering
     ordering_fields = ('total_samples_count', 'id', 'created_at', 'source_first_published', 'accession_code',)
-    samples_count = django_filters.NumberFilter(method='filter_samples_count')
```

Comment:
Why was this removed without comment?

Reply:
I thought it wasn't needed, since we now use the field `total_samples_count`. Sorting by number of samples works locally for me.

I should have added a comment though.